toolkit/analysis_and_visualization/classification/assets/styles/simplePagination.css

```diff
@@ -0,0 +1,282 @@
+
+  /* Opera11.10+ */
+  background: -ms-linear-gradient(top, #bbb 0%, #efefef 100%);
+
+  /* IE10+ */
+  background: linear-gradient(top, #bbb 0%, #efefef 100%);
+
+  /* W3C */
+  cursor: default;
+}
+
+.compact-theme .ellipse {
+  background: #eaeaea;
+  padding: 0 10px;
+  cursor: default;
+}
+
+/* ------------------------------------*\
+	Light Theme Styles
+\*------------------------------------ */
+.light-theme span {
+  cursor: pointer;
+}
+
+.light-theme a,
+.light-theme span {
+  float: left;
+  color: #666;
+  font-size: 14px;
+  line-height: 24px;
+  font-weight: normal;
+  text-align: center;
+  border: 1px solid #bbb;
+  min-width: 14px;
+  padding: 0 7px;
+  margin: 0 5px 0 0;
+  border-radius: 3px;
+  box-shadow: 0 1px 2px rgba(0, 0, 0, 0.2);
+  background: #efefef;
+
+  /* Old browsers */
+  background: -moz-linear-gradient(top, #fff 0%, #efefef 100%);
+
+  /* FF3.6+ */
+  background: -webkit-gradient(linear, left top, left bottom, color-stop(0%, #fff), color-stop(100%, #efefef));
+
+  /* Chrome,Safari4+ */
+  background: -webkit-linear-gradient(top, #fff 0%, #efefef 100%);
+
+  /* Chrome10+,Safari5.1+ */
+  background: -o-linear-gradient(top, #fff 0%, #efefef 100%);
+
+  /* Opera11.10+ */
+  background: -ms-linear-gradient(top, #fff 0%, #efefef 100%);
+
+  /* IE10+ */
+  background: linear-gradient(top, #fff 0%, #efefef 100%);
+
+  /* W3C */
+}
+
+.light-theme a:hover,
+.light-theme li:not(.disabled):not(.active) span:hover {
+  text-decoration: none;
+  background: #fcfcfc;
+}
+
+.light-theme .current {
+  background: #666;
+  color: #fff;
+  border-color: #444;
+  box-shadow: 0 1px 0 rgba(255, 255, 255, 1), 0 0 2px rgba(0, 0, 0, 0.3) inset;
+  cursor: default;
+}
+
+.light-theme .ellipse {
+  background: none;
+  border: none;
+  border-radius: 0;
+  box-shadow: none;
+  font-weight: bold;
+  cursor: default;
+}
+
+/* ------------------------------------*\
+	Dark Theme Styles
+\*------------------------------------ */
+.dark-theme span {
+  cursor: pointer;
+}
+
+.dark-theme a,
+.dark-theme span {
+  float: left;
+  color: #ccc;
+  font-size: 14px;
+  line-height: 24px;
+  font-weight: normal;
+  text-align: center;
+  border: 1px solid #222;
+  min-width: 14px;
+  padding: 0 7px;
+  margin: 0 5px 0 0;
+  border-radius: 3px;
+  box-shadow: 0 1px 2px rgba(0, 0, 0, 0.2);
+  background: #555;
+
+  /* Old browsers */
+  background: -moz-linear-gradient(top, #555 0%, #333 100%);
+
+  /* FF3.6+ */
+  background: -webkit-gradient(linear, left top, left bottom, color-stop(0%, #555), color-stop(100%, #333));
+
+  /* Chrome,Safari4+ */
+  background: -webkit-linear-gradient(top, #555 0%, #333 100%);
+
+  /* Chrome10+,Safari5.1+ */
+  background: -o-linear-gradient(top, #555 0%, #333 100%);
+
+  /* Opera11.10+ */
+  background: -ms-linear-gradient(top, #555 0%, #333 100%);
+
+  /* IE10+ */
+  background: linear-gradient(top, #555 0%, #333 100%);
+
+  /* W3C */
+}
+
+.dark-theme a:hover,
```

Comment:
Codacy found an issue: [Expected selector ".dark-theme a:hover" to come before selector ".compact-theme li:first-child a" (no-descending-specificity)](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6397592)